### ISSUE TITLE
Notifications - Right click or third click marks the unread notification as read without opening it

### ISSUE THREAD

**Issue by sirinebenabbes** [Meeds-io/meeds#1404]
**Steps to reproduce**

1. Open full screen notification center
2. Open the unread notification filter of a category (tasks for example)
3. Do a third click or right click on a notification

**(optional) Affected version**
6.5

**Current behavior**

The notification is marked as read without opening it.

**Expected behavior**

On third click the notification is marked as read and a new tab opens

On right click the notification is not marked as read
 
Tribe bug:  https://community.exoplatform.com/portal/dw/tasks/taskDetail/67802

**Comment by srenault-meeds** [comment]
> On right click the contextual menu opens, the notification is not marked as read unless one of the options 'open link' is clicked

this one is an enhancement and won't be done now.
Only the bug regarding the "Third click" will be checked to be fixed and the fact that when right-clicking, the notif is considered as read.

Please add a new issue for the enhancement.

cc @Julien-Dubois-eXo

**Comment by sirinebenabbes** [comment]
@srenault-meeds I edited the description of the expected behavior related to 

> On right click the contextual menu opens, the notification is not marked as read unless one of the options 'open link' is clicked

and opened its PFR https://github.com/Meeds-io/meeds/issues/1406

**Comment by srenault-meeds** [comment]
FYI @sirinebenabbes @Julien-Dubois-eXo 
We have reviewed 2 items here:
- We tried to improve the "third-click" behaviour. It looks better. Now, you might see sometimes that you still need to click twice to open the notif in a new tab using this "third-click". This is related to the browser itself and we won't be able to do anything more for now.
- We have enhanced the UX when using the "right-click" so the user gets the contextual menu for a notif and thus, we have added a new option in the menu "Open in a new tab".  This was identified here and it is now enhanced https://github.com/Meeds-io/meeds/issues/1406

See below for right click UX:
![rightclick to notif](https://github.com/Meeds-io/meeds/assets/58853195/ddec36e7-040f-41a2-8d38-5f45ba13d11a)
